src/bin/ion/commands/generate/model.rs

```diff
@@ -265,16 +266,12 @@ pub struct WrappedScalar {
     //   type: string
     // }
     // ```
-    // Corresponding `FullyQualifiedTypeReference` would be as following:
-    // ```
-    // FullyQualifiedTypeReference {
-    //    type_name: vec!["Foo"], // name of the wrapped scalar type
-    //    parameters: vec![FullyQualifiedTypeReference {type_name: vec!["String"] }] // base type name for the scalar value
-    // }
-    // ```
-    name: FullyQualifiedTypeReference,
+    // Corresponding `name` would be `vec!["Foo"]` and `base_type` would be `FullyQualifiedTypeReference { type_name: vec!["String"], parameters: vec![] }`.
```

Comment:
I think the comments that were removed were helpful

Reply:
So the previous comments reflected to previous properties and this new only explains the modified properties.